content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
+
+{{< command >}}
+cp java_home/lib/security/cacerts /tmp/kafka.client.truststore.jks
+{{< / command >}}
+
+While still in the bin folder of the Apache Kafka installation on the client machine, create a text file named `client.properties` with the following contents.
+
+{{< command >}}
+ssl.truststore.location=/tmp/kafka.client.truststore.jks
+{{< / command >}}
+
+Run the following command, replacing ```ClusterArn``` with the Amazon Resource Name (ARN).
+
+
+{{< command >}}
+awslocal kafka get-bootstrap-brokers --region us-east-1 --cluster-arn ClusterArn
+{{< / command >}}
+
+From the JSON result of the command, save the value associated with the string named "`BootstrapBrokerStringTls`" because you need it in the following commands.
+
+{{< command >}}
+{
+    "BootstrapBrokerString": "localhost:4514"
+}
+{{< / command >}}
+
+Run the following command in the bin folder, replacing `BootstrapBrokerStringTls` with the value that you obtained when you ran the previous command.
+
+{{< command >}}
+./kafka-console-producer.sh --broker-list BootstrapBrokerStringTls --producer.config client.properties --topic LocalMSKTopic
+{{< / command >}}
+
+Enter any message that you want, and press Enter. Repeat this step two or three times. Every time you enter a line and press Enter, that line is sent to your Apache Kafka cluster as a separate message.
+
+Keep the connection to the client machine open, and then open a second, separate connection to that machine in a new window.
+
+In the following command, replace ```BootstrapBrokerStringTls``` with the value that you saved earlier. Then, go to the bin folder and run the command using your second connection to the client machine.
+
+{{< command >}}
+./kafka-console-consumer.sh --bootstrap-server BootstrapBrokerStringTls --consumer.config client.properties --topic LocalMSKTopic --from-beginning
+{{< / command >}}
+
+You start seeing the messages you entered earlier when you used the console producer command. These messages are TLS encrypted in transit.
+
+Enter more messages in the producer window, and watch them appear in the consumer window.
+
+## Delete the local MSK cluster
+
+Run the following command to list your msk clusters
+
+{{< command >}}
+awslocal kafka list-clusters --region us-east-1
+{{< / command >}}
+
+From the list of clusters, pick the ```ClusterARN``` of the cluster you want deleted and run the command
+
+{{< command >}}
+awslocal kafka delete-cluster --region us-east-1 --cluster-arn ClusterArn
+{{< / command >}}
+
+## Local MSK and Lambdas
+
+### Adding a local MSK trigger 
+
+The following example uses the create-event-source-mapping awslocal command to map a Lambda function named my-kafka-function to a Kafka topic named LocalMSKTopic. The topic's starting position is set to LATEST.
```

Comment:
`uses the create-event-source-mapping awslocal command` -> `uses the Lambda Event Source Mapping API`

Reply:
Done